docs/sources/flow/reference/components/loki.source.api.md

```diff
@@ -0,0 +1,93 @@
+---
+title: loki.source.api
+---
+
+# loki.source.api
```

Comment:
Let's find some place to mention what the two exposed endpoints are and what different purpose they serve as currently people wouldn't know where to send logs to.

Reply:
There's a very concise definition [here](https://grafana.com/docs/loki/next/clients/promtail/configuration/#loki_push_api)

```
Promtail also exposes a second endpoint on /promtail/api/v1/raw which expects newline-delimited log lines. This can be used to send NDJSON or plaintext logs.
```

Also, from the codebase I think we could say that the raw endpoint does not honor `use_incoming_timestamp`.

Reply:
That will do. Thanks!